### PR TITLE
fixing user not being able to enter value as per original bootstrap component

### PR DESCRIPTION
**Pull request by malliapi**
As per original component http://eonasdan.github.io/bootstrap-datetimepicker/#minimum-setup the user is able to edit the input field and correct the date.

We were not able to implement that as the code filters out the invalid dates before calling the onChange method.

Is it possible to let me know if we can make this change ASAP?

### CODE REVIEW

**Comment by malliapi** [comment]
Any news?

**Comment by chollier** [comment]
Hey @malliapi,

Thanks for contributing, I've been willing to implement that feature. However I tried your branch but I couldn't get it working with the expected behavior when running your branch. 
Did you try running the examples ? `http://localhost:8080/webpack-dev-server/basic/index.html`

File: src/DateTimeField.jsx [hunk 1]
```diff
@@ -38,35 +42,42 @@ DateTimeField = React.createClass({
...
+    }, function() {
+      return this.props.onChange(this.state.selectedDate.format(this.props.format));
+    });
+    
   },
   setSelectedDate: function(e) {
+
     return this.setState({
-      selectedDate: this.state.viewDate.clone().date(parseInt(e.target.innerHTML)).hour(this.state.selectedDate.hours()).minute(this.state.selectedDate.minutes())
+      selectedDate: moment().date(parseInt(e.target.innerHTML)).hour(this.state.selectedDate.hours()).minute(this.state.selectedDate.minutes())
```
Comment (chollier) [inline src/DateTimeField.jsx]: doing that will result on the overlay to always display January 70

**Comment by chollier** [comment]
I've created another PR that — I think — fixes the behavior https://github.com/quri/react-bootstrap-datetimepicker/pull/22/files

**Pull request closed without merging**